runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1346,8 +1346,9 @@ NNFW_STATUS nnfw_session::train_prepare()
 
   try
   {
-    if (not _train_info->isValid())
-      throw std::runtime_error{"training info is not valid"};
+    const auto res = _train_info->isValid();
+    if (not res.valid)
```

Comment:
Just a note. Runtime have not used `not`, we've used `!` so far. I am okay for using alternative tokens.